src/main/java/org/jenkinsci/plugins/workflow/cps/CpsHttpFlowDefinition.java

```diff
@@ -0,0 +1,158 @@
+/*
+ * The MIT License
+ *
+ * Copyright 2019 CloudBees, Inc.
```

Comment:
Needs to be changed if you do not work for CloudBees

Reply:
I just write my name or nothing at all?

Reply:
Your name, your company name, or whatever "Plugin contributors" if you prefer.